tests/test_io.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import pandas as pd
+import pytest
+import xarray as xr
+import xclim.xclim.sdba.measures
```

Comment:
```suggestion
import xclim.sdba.measures
```

Reply:
actually, I think this i never used. I would just remove the line.

Reply:
Good catch, removed.